sdk/eventhubs/azure_messaging_eventhubs/src/common/mod.rs

```diff
@@ -53,17 +52,17 @@ impl ManagementInstance {
             .call(EVENTHUB_ENTITY_TYPE.to_string(), application_properties)
             .await?;
 
-        if !response.contains_key(EVENTHUB_PROPERTY_PARTITION_COUNT.to_string()) {
+        if !response.contains_key(&EVENTHUB_PROPERTY_PARTITION_COUNT.to_string()) {
```

Comment:
Aren't these already defined as `&'static str`? Why allocate a string just to take an address? Just pass the symbol:

```suggestion
        if !response.contains_key(EVENTHUB_PROPERTY_PARTITION_COUNT) {
```

Clippy should've warned you about this. Weird.

Reply:
This is weird. On my view of the source, line 55/56 doesn't contain the & and the to_string - it's just your suggested change.